.github/workflows/ci.yml

```
@@ -0,0 +1,17 @@
+name: CI
+
+on: push
+
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: jwlawson/actions-setup-cmake@v1.13
+        with:
+          cmake-version: 3.x
+      - run: mkdir build
+      - run: cmake ..
+        working-directory: build
+      - run: make VERBOSE=1
+        working-directory: build
```

I believe this is the recommended way of building with CMake 3 nowadays:
```suggestion
      - run: cmake -S . -B build
      - run: cmake --build build -v
```